doc/odoc_for_authors.mld

```diff
@@ -541,6 +541,74 @@ in block form this becomes:
 See the {{:https://katex.org/docs/supported.html}KaTeX documentation} for the
 HTML mode LaTeX support status.
 
+{2 Tables}
+
+Odoc 2.3 introduced new markup for tables. This markup comes in two flavors: the light syntax, and the heavy syntax.
+
+The heavy syntax uses several markup: [{table ...}] to define a table, [{tr ...}] to define a line, and [{th ...}] and [{td ...}] to respectively define a header cell and a normal cell.
```

Comment:
to define a line -> to define a row 
a normal cell -> a data cell

(makes it easier understand what `tr` and `td` could stand for)

Reply:
Thanks, good point!